search-enterprise/latest/admin/index.md

```diff
@@ -0,0 +1,372 @@
+* Increase resources for the server if ingestion rate is decreasing with higher volumes of data.
+
+## Permissions indexing performance
+
+### Alfresco Repository
+
+Recommendations:
+
+* Concurrent multiple updates on the same node can lead to missed information on elasticsearch. [SEARCH-2772](https://alfresco.atlassian.net/browse/SEARCH-2772)
+* Updating permissions on folder containing a high number of nodes may cause that the permissions of descendants are not correctly updated on elasticsearch. [SEARCH-2768](https://alfresco.atlassian.net/browse/SEARCH-2768)
+
+## Content indexing performance
+
+### Alfresco Repository
+
+Recommendations:
+
+* Increasing the number of concurrent consumers for the ActiveMQ Queue `acs-repo-transform-request` can lead to missed content transformations. Maintain this value below 10 in order to be sure all the documents are transformed.
+
+### Alfresco Elasticsearch Connector
+
+Recommendations:
+
+* Increase timeout used to contact via HTTP the Shared File Store setting the value in `alfresco.sharedFileStore.timeout`. This will help to avoid issues when the Shared File Store response slower than usual under a huge load.
+* Increase content retry delay used to retrieve a content from Shared File Store setting the value in `alfresco.content.event.retry.delay`. This will help when we are doing a massive document upload in the system and we have to wait for the content available more than usual.
+* Choose the right configuration for the Shared File Store content age scheduler. If for disk space issues you reduce it, consider that you can face many not found exceptions with a consequent content reinsert request.
+
+## Searching performance
+
+Alfresco Elasticsearch Connector is using an out-of-the-box Elasticsearch server. In order to improve query performance, [official recommendations from Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/master/tune-for-search-speed.html) are valid.
+
+## Reindexing
+
+Reindexing component has been tested on reading replica with Postgres database.
+Tests have been performed on the local environment and AWS.
+
+To run database read replicas in AWS follow the guidelines given in [AWS read replicas](https://aws.amazon.com/rds/features/read-replicas/l)
+
+For using the read replica in the reindexing phase, configure the reindexing component for targeting the correct read replica:
+
+```bash
+spring.datasource.url=jdbc:postgresql://<READ_REPLICA_ADDRESS>:<READ_REPLICA_PORT>/alfresco
+```
+
+## Reindex using remote partitioning
+
+When it is required to index a huge Alfresco Repository instance, using a single reindex process can require much time. We can scale the reindex node vertically in order to improve performance, but this cannot be enough. Using [remote partitioning](https://docs.spring.io/spring-batch/docs/current/reference/html/scalability.html#partitioning), a Spring Batch feature, we can scale horizontally the Reindexing service.
+
+
```

Comment:
Following picture need to be formatted as code

Reply:
this page is a part of another PR